vs/dosbox.vcxproj.filters

```diff
@@ -136,6 +136,9 @@
     <ClCompile Include="..\src\hardware\covox.cpp">
       <Filter>src\hardware</Filter>
     </ClCompile>
+    <ClCompile Include="..\src\hardware\compressor.cpp">
+      <Filter>src\hardware</Filter>
+    </ClCompile>
```

Comment:
I think the soft-limiter's unit test .cpp still exists - that could be deleted or repurposed (and renamed) to become the new unit tests for the compressor.
 
It would be nice to test a handful of sunny day and off-cases. Something like:
 - both normal
 - left normal, right out of bounds (pos)
 - left normal, right out of bounds (neg)
 - left out of bounds (pos), right normal
 - left out of bounds (neg), right normal
 - both out of bounds (neg)
 - both out of bounds (pos)
 - both out of bounds (neg + pos)

I think could drop the compress in place of the limiter, and the unit tester will give you back the new numbers (which you can inspect and use as the new truths, if they look roughly correct).

Reply:
Yes, I'll nuke the old unit tests.

To be frank, I'm finding setting up unit test in C++ to be a major pain; at one point I've wasted like 2 hours on it and gave up in frustration (it was an attempt to add tests to `AddSamples`. Maybe it's not the fault of C++, just our project structure makes it more difficult than it needs to be, plus maybe I don't know what I'm doing...

I not really convinced that it's worthwhile to add unit tests to DSP stuff. Not because it's impossible, of course it's possible, but it's just so hard, especially with time-dependent processors like this. I'd just fire up the code and listen to what it's doing, personally, and most audio guys do the same and rarely bother with automated tests... Ultimately, it has to sound good.

Reply:
Yup; it's a major pain when the stuff under tests needs dosbox alive and ticking (or in the mixers case, makes calls to SDL). 

I promise you'll have a very easy time dropping the compressor in place of the limiter, if you want to try. But I'm also confident you've push a lot of audio through it to catch any corner cases too. :👍